src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx

```diff
@@ -60,38 +50,45 @@ const NoLabels = styled.div`
 `;
 
 const NoLabelsEditLabelTooltip = styled(EditLabelTooltip)`
-  margin-left: ${({ theme }) => theme.spacingPx * 2}px;
+  margin-left: ${({ theme }) => theme.global.edgeSize.medium};
+`;
+
+const Dotted = styled.span`
+  border-bottom: 1px dotted;
 `;
 
+interface IClusterLabelsProps
+  extends Omit<ComponentPropsWithoutRef<'div'>, 'onChange'> {
+  onChange: (patch: ILabelChange) => void;
+  labels?: Record<string, string>;
+  isLoading?: boolean;
+  errorMessage?: string;
+  showTitle?: boolean;
```

Comment:
Making the component controllable, so I can re-use it for the MAPI implementation